src/BugsnagUnity/Native/Android/NativeInterface.cs

```diff
@@ -38,10 +38,13 @@ class NativeInterface
     private IntPtr ObjectToString;
 
     private bool CanRunOnBackgroundThread;
+
+    private bool Unity2019OrNewer;
```

Comment:
Can we word this wrt whether `SBytes` is available rather than the Unity version? I think it makes it a little more obvious what we're doing.

```suggestion
    private bool CanUseSBytes;
```

Reply:
Discussed IRL and agreed to leave as-is.